### PR TITLE
add title and description for file formats

### LINKED ISSUE #58: /file_formats title and description
`GET /file_formats` lists all supported file formats, but title and description is missing for each format. We can copy this from Cito if the format is supported, https://openeocloud.vito.be/openeo/1.2/file_formats

### PR DESCRIPTION
**Pull request by zcernigoj**
GET /file_formats lists all supported file formats

this MR adds titles and descriptions for the file formats

`GTIFF`, `NETCDF`, `PNG` : titles and descriptions taken from VITO / other openEO backends
`ZARR`: title is currently just "ZARR", description taken from https://zarr.readthedocs.io/en/stable/
`JPEG`: title is what the JPEG stands for, description is written anew

fixes https://github.com/Open-EO/openeo-sentinelhub-python-driver/issues/58

### CODE REVIEW

File: rest/output_formats/JPEG.json [hunk 1]
```diff
@@ -2,6 +2,8 @@
   "gis_data_types": [
     "raster"
   ],
+  "title":" Joint Photographic Experts Group",
+  "description": "JPEG is a image format that uses lossy compression and is one of the most widely used image formats. Compared to other EO raster formats, it is less flexible and standardized regarding number of bands, embedding geospatial metadata, etc.",
```
Comment (zansinergise) [inline rest/output_formats/JPEG.json]: I would just change to: JPEG is a**n** image format

**Review by zansinergise — CHANGES_REQUESTED** [review]
Looks good otherwise, except for the typo

File: rest/output_formats/JPEG.json (hunk 1 above)
Comment (zcernigoj) [reply rest/output_formats/JPEG.json]: Good catch, thanks. Fixed it

**Review by zansinergise — APPROVED** [review]
Looks good 🥇

**Pull request merged**